static/app/views/onboarding/targetedOnboarding/components/integrationSelect.tsx

```diff
@@ -12,7 +12,7 @@ type Integration = {
   slug: string;
 };
 // TODO: We might want to put this on the backend eventually for centralized source of truth.
-const availabledIntegrations: Integration[] = [
+export const OnboardingIntegrations: Integration[] = [
```

Comment:
Should this be capitalized like this, or should it be `ONBOARDING_INTEGRATIONS`. Capitalization is usually reserved for classes and functions